src/button/styles/button.css

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/variables.css';
+
+.button {
+	background-color: var(--button-bg);
+	border: 1px solid var(--button-bg);
+	box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.25);
```

Comment:
This and most other numerical values below (`border-radius`, `padding`, `transition-duration`, `font-size`, etc.) should be CSS variables.